server/src/test/java/org/elasticsearch/action/admin/indices/close/CloseIndexResponseTests.java

```diff
@@ -113,7 +113,9 @@ protected void assertEqualInstances(CloseIndexResponse expected, CloseIndexRespo
                             assertThat(actualFailure.getNodeId(), equalTo(expectedFailure.getNodeId()));
                             assertThat(actualFailure.index(), equalTo(expectedFailure.index()));
                             assertThat(actualFailure.shardId(), equalTo(expectedFailure.shardId()));
-                            assertThat(actualFailure.reason(), equalTo(expectedFailure.reason()));
+                            // Serialising and deserialising an exception seems to remove the "java.base/" part from the stack trace,
+                            // so these string replacements account for this.
+                            assertThat(actualFailure.reason().replace("java.base/", ""), equalTo(expectedFailure.reason().replace("java.base/", "")));
```

Comment:
I made this change (and again in 1 other class) to get the test to pass, but I don't like it. I'd appreciate other suggestions.